src/test/java/se/iths/librarysystem/entity/GenreEntityTest.java

```diff
@@ -4,7 +4,7 @@
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
```

Comment:
This seems to be another issue. Am I missing something?

Reply:
I have merged `main` into this branch 3 times and I think this is a change coming from `main` branch.